debian/changelog

```diff
@@ -2,7 +2,7 @@ daos (1.3.0-3) unstable; urgency=medium
   [ Alexander Oganezov ]
   * Update to ofi v1.12.0 release
 
- -- Alexander Oganezov <alexander.a.oganezov@intel.com> Wed, 10 Mar 2021 05:00:00 -0000
+ -- Alexander Oganezov <alexander.a.oganezov@intel.com> Wed, 10 Mar 2021 05:00:01 -0000
```

Comment:
Why was this necessary?  In any case, if you are going to update a timestamp, making it more current would be useful.

Reply:
this was just to retrigger build

Reply:
I wondered if that was it.  `git commit --allow-empty` also works for that.  Or you can just do a _Build with Parameters_ right on the classic view page for the PR in Jenkins.